libs/SalesforceSDKCore/SalesforceSDKCore/Classes/UserAccount/SFUserAccountManager.m

```diff
@@ -1517,7 +1525,7 @@ - (void)setCurrentUserInternal:(SFUserAccount*)user {
             for (SFUserAccountIdentity *identity in keys) {
                 // Logout any other user with Biometric Authentication
                 if ([bioAuthManager checkForPolicyWithUserId:identity.userId] && ![identity isEqual:[self currentUserIdentity]]) {
-                    [self logoutUser:[self userAccountForUserIdentity:identity]];
+                    [self logoutUser:[self userAccountForUserIdentity:identity] reason:SFLogoutReasonUnknown];
```

Comment:
I think this is a candidate for a new logout reason in the future.  It's a stretch but does `SFLogoutReasonUserInitiated` make any sense?  The user is choosing to login again (instead of biometric) and we are revoking the old one because we have a new one.

Reply:
Actually the `REFRESH_TOKEN_ROTATED` seems to fit.

Reply:
I can't remember, do we tell them that's what's happening? I was initially thinking user initiated would be them clicking a log out button but I could see it here if they're choosing to proceed knowing a logout would happen

Reply:
Yeah, user initiated isn't the right fit.  However I think `REFRESH_TOKEN_ROTATED` is perfect.

Reply:
It looks like that one is used in server side logs, I'm wondering if we can use it too, can double check tomorrow

Reply:
Updated!